doc/Caliptra.md

```diff
@@ -583,11 +583,13 @@ Caliptra RT generates the DPE certificate and endorses it with the Alias<sub>RT<
 
 **Definitions**
 
-* **Non-Debug:** Caliptra JTAG is not open for microcontroller and HW debug.
-* **Debug:** Caliptra JTAG is open for microcontroller and HW debug.
+* **Non-Debug:** Caliptra JTAG is not open for microcontroller and HW debug. Alias: DebugLocked
+* **Debug:** Caliptra JTAG is open for microcontroller and HW debug. Alias: DebugUnlocked
 * **Unprovisioned:** Blank/unprogrammed fuse part.
 * **Manufacturing:** Device is in the manufacturing flow where HVM Caliptra fuses are programmed.
 * **Production:** All of Caliptra’s HVM fuses are programmed.
+* **Secure:** Any security state that restricts access to Caliptra from an external entity and has a known, specified function in the device lifecycle. Requires DebugLocked and either the Manufacturing or Production configuration.
+* **Insecure:** Any security state other than those deemed secure. This includes all undefined states, all states with DebugUnlocked, and the Unprovisioned state. In accordance with the [threat model](#threat-model), these states are considered a potential risk due to attacks via DFT or DFD channels, exploitation of unforeseen logic issues, or undefined behavior. Necessitates flushing of Caliptra secrets.
```

Comment:
recommend for last sentence: These insecure states require flushing of Caliptra secrets.

Reply:
Applied